api/src/certification/scoring/infrastructure/repositories/scoring-configuration-repository.js

```diff
@@ -27,7 +27,7 @@ export const getMostRecentByDateAndLocale = async ({ locale, date }) => {
   });
 };
 
-export const save = async (configuration) => {
+export const saveCompetenceForScoringConfiguration = async (configuration) => {
```

Comment:
On ne souhaite plus creer de methode de repo sans destructuration. Cela permet de pouvoir injecter des api externes si besoin
```suggestion
export const saveCompetenceForScoringConfiguration = async ({ configuration }) => {
```